ntoskrnl/ke/amd64/cpu.c

```diff
@@ -89,25 +86,25 @@ KiGetCpuVendor(VOID)
     /* Now check the CPU Type */
     if (!strcmp((PCHAR)Prcb->VendorString, CmpIntelID))
     {
-        return CPU_INTEL;
+        Prcb->CpuVendor = CPU_INTEL;
```

Comment:
Interesting: the `CpuVendor` field already existed for x64 in NT 5.2, but it had to wait until NT 6.0 to appear in the x86 KPRCB: this should explain why our x86 `KiGetCpuVendor()` doesn't assign anything to this field.

Reply:
Yes, I noticed that too.